modules/cnf-intel-n3000-optimize-data-plane.adoc

```diff
@@ -0,0 +1,36 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-intel-n3000-optimize-data-plane_{context}"]
+= Understanding Intel hardware accelerator cards for OpenShift Container Platform
```

Comment:
s/OpenShift Container Platform/{product-title}/

AFAIK, we can use attributes in titles and have several instances in the doc repository.